src/containerapp/azext_containerapp/_utils.py

```diff
@@ -155,23 +155,25 @@ def parse_service_bindings(cmd, service_bindings_list, resource_group_name, name
     for service_binding_str in service_bindings_list:
         parts = service_binding_str.split(",")
         arg_dict = {}
+        service_binding = parts[0].split(':')
+        service_name = service_binding[0]
+
+        if len(service_binding) == 1:
+            binding_name = service_name
+        else:
+            binding_name = service_binding[1]
 
         for part in parts:
             key_value = part.split("=")
-
+            clientType=binding_name+"_clientType"
             if len(key_value) == 1:
                 # This means we don't have comma separated args
-                pass
+                arg_dict[clientType] = "none"
```

Comment:
as `parse_service_bindings` is used by `create` and `update` for `--bind`, when used by update, if always set default value of clientType to 'none', it will modify the value of existing value even the Cx not input the clientType.
 
In CLI, for update, it will infer the `clientType` from existing value if not input.

Reply:
In this PR, If a Cx's binding original clientType is `dotnet`, when it execute with command `az containerapp update` without `clientType`, its clientType will be update to `none`, right?

Reply:
In the line 1125 in `containerapp_decorator.py`, it's the logic about inferring the `clientType` from existing value if not input.
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/c5443785-979d-4f1a-9a83-8c1f28a744b1)